telemetry/packages/constants/src/pods/modes.ts

```diff
@@ -0,0 +1,27 @@
+import { ACTIVE_STATES, PodStateType } from './states';
+
+export type ModeType = keyof typeof MODES;
```

Comment:
probably best this type be below `MODES` which it infers from (just flows better)